assets/css/app.scss

```diff
@@ -1,21 +1,24 @@
 ---
-# We need this frontmatter for Jekyll to compile the less code
+# We need this frontmatter for Jekyll to compile the Sass code
 ---
 
 // Import variables
-@import "bootstrap/less/bootstrap.less";
 @import "user10/variables";
 @import "app-variable-overrides";
+@import "bootstrap/stylesheets/bootstrap";
+
 @import "user10/utilities";
+
+
 @import "vertical-carousel";
 @import "lity";
 
-// Import global styles
-@import "global";
-
 // Import deployment guide styles
 @import "guides";
 
+// Import global styles
+@import "global";
```

Comment:
Ah nice, this ordering switch probably should have been done when we introduced guides.

Reply:
I'm actually swapping their places again - I can see the selectors in `guides.scss` are supposed to override the selectors in `global.scss`.